### PR TITLE
README update with clone instructions

### PR DESCRIPTION
**Pull request by adecaro**
Signed-off-by: Angelo De Caro <adc@zurich.ibm.com>

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,19 @@ Fabric Smart Client has not been audited and is provided as-is, use at your own
 - [`Fabric Token SDK`](https://github.com/hyperledger-labs/fabric-token-sdk): Do you want to develop Token-Based Distributed
 application with simplicity and joy? Check our Token SDK out [`here`](https://github.com/hyperledger-labs/fabric-token-sdk).
 
+# Getting started
+
+Clone the code and make sure it is on your `$GOPATH`. 
+(Important: we assume in this documentation and default configuration that your `$GOPATH` has a single root-directoy!). 
+Sometimes, we use $FSC_PATH to refer to the Fabric Smart Client repository in your filesystem.
```
Comment (mbrandenburger) [inline README.md:32]:
```suggestion
Sometimes, we use `$FSC_PATH` to refer to the Fabric Smart Client repository in your filesystem.
```

File: README.md [hunk 2]
```diff
@@ -25,6 +25,19 @@ Fabric Smart Client has not been audited and is provided as-is, use at your own
...
+
+```bash
+export $FSC_PATH=$GOPATH/src/github.com/hyperledger-labs/fabric-smart-client
+git clone https://github.com/hyperledger-labs/fabric-smart-client.git $FSC_PATH
+```
+
+You are ready to run the [`samples`](./samples/README.md).
```
Comment (mbrandenburger) [inline README.md:39]:
```suggestion
You are ready to run the [`samples`](./samples/README.md) in `$FSC_PATH`.
```

File: samples/README.md [hunk 3]
```diff
@@ -2,9 +2,9 @@
 
 Samples are a collection of small and simple apps that demonstrate how to use the library.
 
-To run the samples, we recommend to use go 1.16 or later. You will also need docker when using Fabric. 
+To run the samples, we recommend to use `go 1.16` or later. You will also need docker when using Fabric.
```
Comment (mbrandenburger) [inline samples/README.md:5]:
```suggestion
To run the samples, we recommend to use `go 1.16` or later. You will also need docker as it is needed by Fabric. 
```

File: samples/README.md [hunk 4]
```diff
@@ -2,9 +2,9 @@
 
 Samples are a collection of small and simple apps that demonstrate how to use the library.
 
-To run the samples, we recommend to use go 1.16 or later. You will also need docker when using Fabric. 
+To run the samples, we recommend to use `go 1.16` or later. You will also need docker when using Fabric. 
 To make sure you have all the required docker images, you can run `make docker-images` in the 
-folder `$GOPATH/src/github.com/hyperledger-labs/fabric-smart-client`.
+folder `$FSC_PATH`.
```
Comment (mbrandenburger) [inline samples/README.md:7]:
```suggestion
Fabric Smart Client repository folder (`$FSC_PATH`).
```

**Review by mbrandenburger — COMMENTED** [review]
Looks good. A few suggestions.

**Comment by adecaro** [comment]
need to close this and reopen another one because I cannot push here :(

**Pull request closed without merging**